connection-api/src/main/java/org/terracotta/connection/entity/EntityRef.java

```diff
@@ -26,27 +26,48 @@
 
 
 /**
- * The EntityRef is a reference to where an entity could potentially exist.  That is, it is created to refer to a specific
- * entity type and name, although there may not yet be an entity with that type and name.
- * 
- * The EntityRef is used to manage the server-side entity's life cycle in that it is how the entity is either created or
- * destroyed.
- * 
- * It is also used as the path through which an actual instance can be fetched for use on the client.
- * 
+ * The EntityRef is a reference to where an entity could potentially exist.
+ * <P>
+ *   That is, it is created to refer to a specific entity type and name, although there may not yet be an entity with
+ *   that type and name.
+ * </P>
+ * <P>
+ *   The EntityRef is used to manage the server-side entity's life cycle in that it is how the entity is either created
+ *   or destroyed.
+ * </P>
+ * <P>
+ *   It is also used as the path through which an actual instance can be fetched for use on the client.
+ * </P>
+ *
  * @param <T> The entity type underlying this reference.
  * @param <C> The configuration type to use when creating this entity.
  */
 public interface EntityRef<T extends Entity, C> {
   /**
-   * Create the entity with the given configuration.
+   * Creates the entity with the given configuration.
+   * <P>
+   *   If the entity already exists, the {@link EntityAlreadyExistsException} is  thrown immediately, whether or not
```

Comment:
Condition that indicates we do not want the operation to block when the entity is already fetched on any client.